src/software/ai/hl/stp/skill/pivot_kick/pivot_kick_skill_fsm.cpp

```diff
@@ -24,13 +24,14 @@ void PivotKickSkillFSM::kickBall(const Update& event)
 
 bool PivotKickSkillFSM::lostPossession(const Update& event)
```

Comment:
I would call this `robotLostLocalBallTraction` or something similar to not confuse with the case when the team turns over possession of the ball to the enemy team and they have control over it. maybe we can discuss different levels of control/possession as it will probably be useful in `PlayMonitor` too.